test/fixture/composition-api/pages/index.vue

```diff
@@ -0,0 +1,17 @@
+<template>
+  <div>
+    <div id="current-locale">locale: {{ $i18n.locale }}</div>
+    <div id="content">{{ someText }}</div>
+  </div>
+</template>
+
+<script>
+import { defineComponent, useContext } from '@nuxtjs/composition-api'
+export default defineComponent({
+  setup () {
+    const { app: { i18n } } = useContext()
+    const someText = i18n.t('untranslated')
```

Comment:
Let's use an existing string so that we can check that it translates also:
```suggestion
    const someText = i18n.t('home')
```
That should translate to `Homepage`.

Reply:
Good point. Changed this